main/src/ca/uwaterloo/flix/language/phase/Weeder.scala

```diff
@@ -748,7 +748,7 @@ object Weeder {
 
           case ("SCOPE_EXIT", e1 :: e2 :: Nil) => WeededAst.Expression.ScopeExit(e1, e2, loc).toSuccess
 
-          case _ => WeederError.IllegalIntrinsic(loc).toFailure
+          case _ => Validation.SoftFailure(WeededAst.Expression.Wild(loc), LazyList(IllegalIntrinsic(loc)))
```

Comment:
Why does it become WeededAst.Expression.Wild? Should it not be Expression.Error?

Reply:
Oh yes of course! I took too much time off :D